functions/README.md

```diff
@@ -0,0 +1,48 @@
+# Installation Guide
+
+1. Create a budget in the `Budgets & alerts` tab <https://console.cloud.google.com/billing>
+
+1. Enable `Connect a Pub/Sub topic to this budget`
+
+1. Create a `Cloud Pub/Sub topic` for the firebase project and name it `billing`
+
+1. Enable Cloud Billing API <https://console.developers.google.com/apis/api/cloudbilling.googleapis.com>
+
+1. Give billing administrative privilege to <PROJECT_ID>@appspot.gserviceaccount.com <https://console.cloud.google.com/billing>
+
+1. Change the project id in `.firebaserc` and `index.js`
+
+1. Set up firebase cli `npm install -g firebase-tools`
```

Comment:
Might be better to stick to yarn commands to be consistent with the rest of the repo eg. `yarn global add firebase-tools`. Also `yarn install` and `yarn add @google-cloud...`down below

Reply:
maybe it is just me but there were a lot of deployment errors when i used yarn. I test the deployment again using yarn this time.